pkg/operator/ceph/cluster/cephstatus_test.go

```diff
@@ -91,6 +91,21 @@ func TestCephStatus(t *testing.T) {
 	assert.Equal(t, osdDownMsg.Severity, aggregateStatus.Details["OSD_DOWN"].Severity)
 	assert.Equal(t, pgAvailMsg.Summary.Message, aggregateStatus.Details["PG_AVAILABILITY"].Message)
 	assert.Equal(t, pgAvailMsg.Severity, aggregateStatus.Details["PG_AVAILABILITY"].Severity)
+
+	// Test for storage capacity of the ceph cluster
+	newStatus = &cephclient.CephStatus{
```

Comment:
Please comment on your tests to explain what you are testing. Also, you need to test the failure scenario and make sure we keep the existing values. Just like mentioned here https://github.com/rook/rook/pull/6475/files#r510891005

Reply:
Done.